### DISCUSSION TITLE
Keyball61 特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by backclover** [Yowkees/keyball#570] [Q&A]
お世話になっております。
先日Keyall61を購入し、組み立てましたがキー2箇所が反応しません。
お手数おかけしますが、助言いただけますと幸いです。

- 反応しないのは、右手側(トラックボール)の左端２箇所です。(添付画像参照)
- キーソケットとダイオードは再はんだ付けしました。
- キーソケット接続部分をショートさせても反応しないです。

よろしくお願いいたします。

![IMG_3136](https://github.com/Yowkees/keyball/assets/69007174/a0a0f6ed-a3be-454c-b501-eda96f16ee73)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

添付させていただいた写真の緑の丸で示しましたダイオードが反応しないキー一列に対応しているダイオードなのではんだ付けのやり直しか取り換えをお願いいたします。
![327068980-a0a0f6ed-a3be-454c-b501-eda96f16ee73](https://github.com/Yowkees/keyball/assets/143698349/f0fdd75a-4334-41fe-8970-517ec01fa3c5)

**Reply by backclover** [reply]
ありがとうございます。
頂いた情報をもとに、ダイオードを２回新しいものに変更してみましたが、状態変わらずでした。

他に可能性ございましたらご教示いただけますと幸いです。

**Reply by backclover** [reply]
添付画像でそれぞれ結んでいる場所の導通は確認できました。
![IMG_3136dd](https://github.com/Yowkees/keyball/assets/69007174/e539ad5a-d985-4fc1-9f66-1b3cd1782931)
